packages/toggle/src/Toggle.tsx

```diff
@@ -203,59 +213,69 @@ const sizeStyles = {
       ${sliderSelector.checked} {
         transform: translate3d(12px, 0, 0);
       }
-
-      ${sliderSelector.disabled} {
-        height: 10px;
-        width: 10px;
-        left: 1px;
-      }
-
-      ${buttonSelectors.checked}:disabled {
-        transform: translate3d(11px, 0, 0);
-      }
     `,
   },
 } as const;
 
+// TODO: Refresh - remove when darkMode is updated
+const sizeStylesDarkMode: Record<Size, string> = {
+  [Size.Default]: css`
+    ${sliderSelector.disabled} {
+      height: 28px;
+      width: 28px;
+    }
+  `,
+  [Size.Small]: css`
+    ${sliderSelector.disabled} {
+      height: 18px;
+      width: 18px;
+      left: 1px;
+    }
+
+    ${buttonSelectors.checked}:disabled {
+      transform: translate3d(17px, 0, 0);
+    }
+  `,
+  [Size.XSmall]: css`
+    ${sliderSelector.disabled} {
+      height: 10px;
+      width: 10px;
+      left: 1px;
+    }
+
+    ${buttonSelectors.checked}:disabled {
+      transform: translate3d(11px, 0, 0);
+    }
+  `,
+};
+
 const modeStyles = {
   [Mode.Light]: {
     button: css`
-      box-shadow: inset 0 0 5px rgba(6, 22, 33, 0.1);
-
       &[aria-checked='false']:not(:disabled) {
-        background-color: rgba(61, 79, 88, 0.1);
-        border-color: rgba(18, 22, 22, 0.03);
+        background-color: ${palette.gray.base};
+        border-color: ${palette.gray.base};
       }
 
       &:disabled {
-        background-color: rgba(6, 22, 33, 0.09);
-        border-color: rgba(6, 22, 33, 0.04);
-        box-shadow: none;
+        background-color: ${palette.gray.light2};
+        border-color: ${palette.gray.light2};
       }
     `,
 
     slider: css`
-      background-color: white;
-      box-shadow: 0 0 2px rgba(28, 192, 97, 0.08), 0 1px 2px rgba(0, 0, 0, 0.25),
-        inset 0 -1px 0 #f1f1f1;
-
-      &:before {
-        background-image: linear-gradient(${uiColors.white}, #f6f6f6);
-      }
+      background-color: ${palette.white};
+      display: flex;
+      justify-content: center;
+      align-items: center;
 
       ${sliderSelector.disabled} {
-        box-shadow: none;
-        background-color: rgba(6, 22, 33, 0.09);
+        background-color: ${palette.gray.light3};
       }
     `,
 
-    offLabel: css`
-      color: ${uiColors.gray.dark1};
-    `,
-
-    onLabel: css`
-      color: ${uiColors.white};
-    `,
+    offLabel: css``,
+    onLabel: css``,
```

Comment:
Are we just keeping these around for TS, since dark mode needs them? Maybe add a `//TODO` to remove?